roles/openshift_hosted_logging/tasks/deploy_logging.yaml

```diff
@@ -138,7 +138,7 @@
     delay: 10
 
   - name: "Deploy fluentd by labeling the node"
-    shell:  "{{ openshift.common.client_binary }} --config={{ mktemp.stdout }}/admin.kubeconfig label node {{ openshift_hostname }} {{ openshift_hosted_logging_fluentd_nodeselector if openshift_hosted_logging_fluentd_nodeselector is defined else 'logging-infra-fluentd=true' }}"
+    shell:  "{{ openshift.common.client_binary }} --config={{ mktemp.stdout }}/admin.kubeconfig label node --all {{ openshift_hosted_logging_fluentd_nodeselector_label if openshift_hosted_logging_fluentd_nodeselector_label is defined else 'logging-infra-fluentd=true' }}"
```

Comment:
@richm Thoughts on what makes the most sense here? I think we should make the label for the daemon set immutable, or at least an advanced setting that 99% of users wouldn't touch only if it conflicts with a label they're alread using for something else. But we should add a selector to the determine which nodes we apply that label to. The default would be no selector meaning that all nodes would get fluentd label applied to them.